account_cutoff_base/models/account_cutoff.py

```diff
@@ -118,7 +118,7 @@ def _default_cutoff_account_id(self):
     cutoff_account_id = fields.Many2one(
         comodel_name="account.account",
         string="Cut-off Account",
-        domain="[('deprecated', '=', False), ('company_id', '=', company_id)]",
+        domain="[('deprecated', '=', False)]",
```

Comment:
You should keep company in the domain and replace it with ('company_ids', 'in', company_id)
This problem is present at several places in this commit.
So I propose another migration PR: https://github.com/OCA/account-closing/pull/329